website/src/SUMMARY.md

```diff
@@ -44,37 +44,42 @@
     - [config.remove](commands/config-remove.md)
     - [config.setup](commands/config-setup.md)
     - [offline](commands/offline.md)
+- [Configuration file](configuration-file.md)
 - [Preferences](preferences.md)
-  - [bitbucket-app-password](preferences/bitbucket-app-password.md)
-  - [bitbucket-username](preferences/bitbucket-username.md)
-  - [configuration file](configuration-file.md)
-  - [contribution-branches](preferences/contribution-branches.md)
-  - [contribution-regex](preferences/contribution-regex.md)
-  - [create-prototype-branches](preferences/create-prototype-branches.md)
-  - [default-branch-type](preferences/default-branch-type.md)
-  - [dev-remote](preferences/dev-remote.md)
-  - [feature-regex](preferences/feature-regex.md)
-  - [gitea-token](preferences/gitea-token.md)
-  - [github-token](preferences/github-token.md)
-  - [gitlab-token](preferences/gitlab-token.md)
-  - [hosting-origin-hostname](preferences/hosting-origin-hostname.md)
-  - [hosting-platform](preferences/hosting-platform.md)
-  - [main-branch](preferences/main-branch.md)
-  - [new-branch-type](preferences/new-branch-type.md)
-  - [observed-branches](preferences/observed-branches.md)
-  - [observed-regex](preferences/observed-regex.md)
-  - [offline](preferences/offline.md)
-  - [parent](preferences/parent.md)
-  - [parked-branches](preferences/parked-branches.md)
-  - [perennial-branches](preferences/perennial-branches.md)
-  - [perennial-regex](preferences/perennial-regex.md)
-  - [prototype-branches](preferences/prototype-branches.md)
-  - [push-hook](preferences/push-hook.md)
-  - [push-new-branches](preferences/push-new-branches.md)
-  - [ship-delete-tracking-branch](preferences/ship-delete-tracking-branch.md)
-  - [ship-strategy](preferences/ship-strategy.md)
-  - [sync-feature-strategy](preferences/sync-feature-strategy.md)
-  - [sync-perennial-strategy](preferences/sync-perennial-strategy.md)
-  - [sync-prototype-strategy](preferences/sync-prototype-strategy.md)
-  - [sync-tags](preferences/sync-tags.md)
-  - [sync-upstream](preferences/sync-upstream.md)
+  - [Branches]()
```

Comment:
This could also be named "Branch Types".

Reply:
"Branches" fits better with the config file heading `[branches]`.

Reply:
Makes sense.